alfalfa_worker/worker.py

```diff
@@ -290,19 +330,16 @@ def step_sim(self, message_body):
         else:
             site_id = message_body.get('id')
             site_rec = self.ac.mongo_db_recs.find_one({"_id": site_id})
+            self.worker_logger.logger.info('Site record: {}'.format(site_rec))
 
-            # TODO: do we still want default to be 'osm'?
-            sim_type = site_rec.get("rec", {}).get("sim_type", "osm").replace("s:", "")
+            # TODO: Check if simType is defined, error if not.
+            sim_type = site_rec.get("rec", {}).get("simType", "osm").replace("s:", "")
```

Comment:
this was causing simulations to always be OSMs. 😬

Reply:
Whoops.  I think I had wanted to get all message passing to utilize snake case, but never implemented on web side of things.  Good catch